lib/apphook.c

```diff
@@ -151,11 +151,11 @@ construct_nondumpable_logger(msg_fatal);
 void
 app_startup(void)
 {
+  g_thread_init(NULL);
   main_loop_thread_resource_init();
   msg_init(FALSE);
   iv_set_fatal_msg_handler(app_fatal);
   iv_init();
-  g_thread_init(NULL);
   crypto_init();
```

Comment:
I would rather move the mainloop initialization call after g_thread_init()

Reply:
Do you recommend a different order, than 1. g_thread_init(), 2. main_loop_thread_resource_init()?

Reply:
Oh I think I got it: you mean move the calling of `main_loop_thread_resource_init()` instead of the `g_thread_init()`.
Can you explain the reason, please?

Reply:
OK,. I've moved both functions just before `stats_init()`.